app/uploaders/profile_photo_uploader.rb

```diff
@@ -0,0 +1,12 @@
+# encoding: utf-8
+class ProfilePhotoUploader < CarrierWave::Uploader::Base
+  if Rails.env.production? || Rails.env.development?
+    storage :fog
+  else
+    storage :file
+  end
+
+  # def store_dir
```

Comment:
Hmm, why is this commented out? I'd prefer to merge the code if it's all complete and working.